Lib/zoneinfo/__init__.py

```diff
@@ -0,0 +1,29 @@
+__all__ = [
+    "ZoneInfo",
+    "reset_tzpath",
+    "TZPATH",
+    "ZoneInfoNotFoundError",
+    "InvalidTZPathWarning",
+]
+
+from . import _tzpath
+from ._common import ZoneInfoNotFoundError
+
+try:
+    from _czoneinfo import ZoneInfo
```

Comment:
This is a slightly more complicated situation, because `_czoneinfo` is actually parallel to `zoneinfo._zoneinfo`. In the reference implementation, the two are `zoneinfo._zoneinfo` and `zoneinfo._czoneinfo`, but the limitations of our current build system prevent me from doing that here.